contracts/sfc/SFC.sol

```diff
@@ -328,6 +329,8 @@ contract SFC is Initializable, Ownable, Version {
         snapshot.totalSupply = totalSupply;
     }
 
+    // Seal epoch and set validators for the next epoch
```

Comment:
Finish epoch sealing - store validators of the new epoch into a snapshot. Apply minGasPrice calculated in the sealEpoch().